reference/5.1/Microsoft.PowerShell.Core/About/about_Environment_Variables.md

```diff
@@ -231,29 +106,36 @@ The environment variables that store preferences include:
 
 - PSModuleAnalysisCachePath
 
-  PowerShell provides control over the file that is used to cache data about a
-  module, such as the commands it exports.
+  PowerShell provides control over the file that is used to cache data about
+  modules and their cmdlets. The cache is read at startup while searching for a
+  command and is written on a background thread sometime after a module is
+  imported.
 
-  By default on Windows, this cache is normally stored in the file
-  `Microsoft\Windows\PowerShell\ModuleAnalysisCache` within
-  `$env:LOCALAPPDATA`. The cache is typically read at startup while searching
-  for a command and is written on a background thread sometime after a module
-  is imported.
+  Default location of the cache is:
 
-  To change the default location of the cache, set the
-  `$env:PSModuleAnalysisCachePath` environment variable before starting
-  PowerShell. Changes to this environment variable only affects children
-  processes. The value should name a full path (including filename) that
-  PowerShell has permission to create and write files. To disable the file
-  cache, set this value to an invalid location, for example:
+  - `$env:LOCALAPPDATA\Microsoft\Windows\PowerShell`
```

Comment:
Since this doc applies to both Windows and Linux, you should include the default Linux/macOS path as well

Reply:
I have this in the v6+ versions of the article:

  - Windows PowerShell 5.1: `$env:LOCALAPPDATA\Microsoft\Windows\PowerShell`
  - PowerShell 6.0 and higher: `$env:LOCALAPPDATA\Microsoft\PowerShell`
  - Non-Windows default: `~/.cache/powershell`